services/testService/package.json

```diff
@@ -1,26 +1,36 @@
 {
-  "name": "test_service",
-  "version": "1.2.3",
-  "description": "example microservice",
-  "keywords": [
-    "microservice"
-  ],
-  "main": "index.js",
-  "config": {
-    "services": {
-      "resourceConsumption": "singleton",
-      "state": "stateless",
-      "queue": true,
-      "role": {
+    "name": "test_service",
+    "version": "1.2.3",
+    "description": "example microservice",
+    "licence": "commercial",
+    "private": true,
+    "resourceConsumption": "normal",
+    "priority": "normal",
+    "downScale": true,
+    "scope": "back",
+    "role": {
         "name": "user",
         "version": "*"
-      }
-    }
-  },
-  "homepage": "https://www.yupana.com",
-  "licence": "commercial",
-  "files": [
-    "index.js"
-  ],
-  "dependencies": {}
+    },
+    "main": "index.js",
+    "files": [
+        "index.js"
+    ],
+    "dependantPackages": [
+        {
+            "id": "222f55f7-72ef-4fad-81a7-d2a0a869d324",

```

Comment:
The id is the most reliable, but shall we add the name of the package as well ? Ignored, but meaningful for the user.

Reply:
Done